zeroeq/http/httpHandler.h

```diff
@@ -0,0 +1,151 @@
+
+/* Copyright (c) 2016, Human Brain Project
+ *                     Stefan.Eilemann@epfl.ch
+ *                     Daniel.Nachbaur@epfl.ch
+ */
+
+#ifndef ZEROEQ_HTTP_HANDLER_H
+#define ZEROEQ_HTTP_HANDLER_H
+
+#include <zeroeq/log.h>
+#include <zeroeq/uri.h>
+#include <boost/network/protocol/http/server.hpp>
+#include <string>
+#include <zmq.h>
+
+namespace zeroeq
+{
+namespace http
+{
+
+class HTTPHandler;
+typedef boost::network::http::server< HTTPHandler > HTTPServer;
```

Comment:
cppnetlib server has the same HTTP* prefix as the locally defined classes - removing the HTTP prefix from local classes, or renaming this typedef, could help avoid any confusion